content/items/rggen.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "RgGen"
+description: "Code generator for configuration and status registers"
+authors:
+  - Taichi Ishitani
+links:
+  gh?: "rggen/rggen"
```

Comment:
Please, remove the question marks (`?`). Those are used in the template for representing that these fields are optional. But the fields are `gh:`, `docs:`, `licenses:`, etc.